### DISCUSSION TITLE
3.9.0 Reverb slider off-center in MacOS

### DISCUSSION THREAD

**Discussion by rdica** [jamulussoftware/jamulus#2768] [Bug Investigation]
This was first reported to me from an associate using Jamulus 3.9.0 MacOS Monterey. I upgraded my client on MacOS Catalina and see the same thing, the reverb slider is off-center.

No issues on Ubuntu 18.04 with QT 5.9.5.

Jamulus 3.9.0, QT 6.3.1

Catalina:
<img width="176" alt="Screen Shot 2022-08-01 at 3 34 57 PM" src="https://user-images.githubusercontent.com/16609801/182231338-87f4176e-b725-4169-9ba2-21067086ace8.png">

Monterey:
![Screenshot_2022-08-01_15-38-16](https://user-images.githubusercontent.com/16609801/182231587-b3c68199-255a-433f-9f0b-51295c1362e9.png)
.

**Comment by pljones** [comment]
I don't know the MacOS environment at all but I'm assuming this is the main MacOS build not the legacy build, right?  Does the legacy build run at all on those environments and does it show the same behaviour?

**Reply by rdica** [reply]
Correct, main MacOS build.
I installed the legacy dmg, it runs on MacOS Catalina and it does not display the reverb slider off-center. Its QT version is 5.9.9.

**Reply by ann0see** [reply]
I think I saw that too while testing. Probably it's a Qt bug?

**Reply by pljones** [reply]
Which is a pain, as we updated to Qt 6 to _improve_ MacOS support...

**Reply by pljones** [reply]
Has anyone checked the other skins, by the way?  Is it all faders or only the fancy skin?

**Reply by rdica** [reply]
I tried each skin and the reverb slider was off-enter in each.

**Comment by hoffie** [comment]
Opened as #2772 to keep it tracked along with a link to the suspected Qt bug.

**Comment by pljones** [comment]
https://github.com/jamulussoftware/jamulus/issues/2772 closed due to no one following up.  Please feel free to raise a PR referencing that bug if you have a fix.
